test/Microsoft.Dnx.Testing.CommonUtils/TestLogger.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Dnx.Testing
+{
+    internal static class TestLogger
```

Comment:
This is a copy of Logger from Runtime. I'm creating a new one here so that the logging from tests can be controlled by a separate environment variable DNX_TEST_TRACE. This allows for clarity and verbosity control when logging the output for tests.